Source/Scene/Model.js

```diff
@@ -523,8 +524,11 @@ define([
                 }
                 //>>includeEnd('debug');
 
-                this._boundingSphere.radius = (this.scale * Matrix4.getMaximumScale(this.modelMatrix)) * this._initialRadius;
-                return this._boundingSphere;
+                var scale = (this.scale * Matrix4.getMaximumScale(this.modelMatrix));
+                var scaledBoundingSphere = this._scaledBoundingSphere;
+                scaledBoundingSphere.center = Cartesian3.multiplyByScalar(this._boundingSphere.center, scale, scaledBoundingSphere.center);
```

Comment:
I don't think we should use the scalar `scale` here.  We should use `Matrix4.getScale`.  Can discuss in person.

We should also do some testing with a `modelMatrix` with a scale and non-uniform scale.